src/main/scripts/README.md

```diff
@@ -0,0 +1,53 @@
+# A/V Pairtree Scripts
+
+Various scripts for use with the A/V Pairtree application.
+
+## avpt_data_pipeline.sh
+
+This script constructs a data processing pipeline consisting of A/V Pairtree, Metagetter, and Festerize (in that order), in which the output CSV files of each component application are passed to the next one for further processing.
+
+### Installation
+
+Dependencies:
+- GNU bash (written for version 4.2.46(2)-release (x86_64-redhat-linux-gnu))
+- GNU coreutils
+- curl
+- ffmpeg
+- [inotifywait](https://github.com/inotify-tools/inotify-tools)
+- [UCLALibrary/services-metagetter](https://github.com/UCLALibrary/services-metagetter)
+- [UCLALibrary/festerize](https://github.com/UCLALibrary/festerize)
+
+### Usage
+
+The following environment variables must be set:
+
+Environment variable|Description
+---|---
+AVPTDP_INPUT_DIRECTORY|directory where A/V Pairtree puts .out files; this is the input directory for the pipeline (and thus, for Metagetter)
+AVPTDP_FESTERIZE_OUTPUT_DIRECTORY|directory where Festerize puts .csv files
+AVPTDP_METAGETTER_MEDIA_DIRECTORY|directory where Metagetter will search for A/V media files
+AVPTDP_METAGETTER_OUTPUT_DIRECTORY|directory where Metagetter puts .out files (which are then renamed as .csv); this is the input directory for Festerize
```

Comment:
`Metagetter` outputs files as .csv, not as .out (unless Kevin made a tweak I missed)

Reply:
I did not change that in metagetter. av-pairtree outputs .out though.

Reply:
The version of Metagetter that I have (a945b6a) does not specifically write output files as .csv; it uses whatever extension the input file had. In this case, since A/V Pairtree outputs .out files (which Metagetter then reads), then Metagetter outputs .out files as well.

Reply:
Agreed Metagetter doesn't change file name/extension... was going to object that Metagetter shouldn't be processing anything, since it filters for files with .csv extension; but it only does that for a directory argument--if passed a single file, it just processes it without checking type (which might be an error with Metagetter's design...); and you're passing files one at a time through the chain, right?

Reply:
Yeah, this script passes one file at a time